src/test/java/uk/gov/hmcts/reform/civil/utils/PartyUtilsTest.java

```diff
@@ -169,4 +172,52 @@ void shouldReturnEmpty_whenPartyTypeIsNotIndividualOrSoleTrader(Party.Type type)
             assertThat(PartyUtils.getDateOfBirth(party)).isEmpty();
         }
     }
+
+    @Nested
+    class PartyReferences {
+
+        @Test
+        void shouldReturnReferences_whenNoRefsAvailable() {
+            CaseData caseData = CaseDataBuilder.builder().build();
+
+            String partyReferences = PartyUtils.buildPartiesReferences(caseData);
+
+            assertEquals("", partyReferences);
+        }
+
+        @Test
```

Comment:
maybe shouldReturnReferences_whenCaseHasOneDefendantSolicitor()?

Reply:
Are you suggesting a new test or renaming this one on line 189 (shouldReturnReferences_whenNot1v2DiffSolicitorAndBothRefAvailable)? The shouldReturnReferences_whenNot1v2DiffSolicitorAndBothRefAvailable test tests possibly 1v1 and both refs available. I named this scenario this way to signify that this works for 1v1, 2v1, 1v2 same solicitor (basically Not 1v2 diff solicitor).

Reply:
The name of test has been changed